server/src/models.js

```diff
@@ -10,6 +10,8 @@ let modelMap = null,
 exports.modelMap = modelMap;
 exports.metaMap = metaMap;
 
+let shot_model = require("../../addon/dist/lib/shot.js");
```

Comment:
underscores!  Also since it only really exports AbstractShot, better `const {AbstractShot} = require("...")` ?